.github/workflows/python-app.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       matrix:
         os: ["ubuntu-22.04"]
 #        os: ["ubuntu-22.04", "ubuntu-latest", "windows-latest"]
-        python-version: ["3.9"]
+        python-version: ["3.10"]
```

Comment:
@rogervankanten if the version supported moves to be >=3.10 this should be reported in the readme. Alternatively, it could be good to test also with previous python versions